app/src/main/java/com/github/geohunt/app/utility/DateFormatUtils.kt

```diff
@@ -0,0 +1,49 @@
+package com.github.geohunt.app.utility
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+
+/**
+ * Utility class that proposes static method for different formatting of LocalDateTimes
+ */
+object DateFormatUtils {
+    private val dateFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy")
+
+    /**
+     * Formats given LocalDateTime into a string
+     * Only takes year, month and day into account
+     * Formats using the following pattern "day/month/year"
+     * @param dateTime dateTime to format
+     */
+    fun formatDate(dateTime: LocalDateTime): String {
+        return dateFormatter.format(dateTime)
+    }
+
+    /**
+     * Formats remaining time from the current time (in the sense of LocalDateTime.now()) to the
+     * time given as an argument
+     * The output depends on the difference in time, it will first try to give the difference in days
+     * and will give it in hours if less than a day is left. If less than an hour remains it will
+     * simply return the string "Less than 1 hour"
+     * If argument is null it returns the string "NEVER"
+     * @param to LocalDateTime to which the remaining time should be computed
+     */
+    fun formatRemainingTime(to: LocalDateTime?): String {
+        when(to) {
+            null -> return "NEVER"

```

Comment:
Nitpicking here, but `to` should probably be either non-nullable, or the string returned should be Pascal cased as `Never` instead.

Reply:
I think we talked about having challenges that don't expire and that they would have a null expirationDate. I can still change the message to "Never" or move this special case in the Challenge class itself as it would maybe make more sense (?).

Reply:
Ok, then `null` makes sense to me, but it would be good to specify it somewhere in the documentation that it is its only purpose. I also would advocate for `Never` instead of `NEVER` to keep it visually coherent.

Reply:
I agree with @dhalilov. Even if I think that it's a matter of personal preference, using "Never" instead of "NEVER" does make it visually coherent. Great discussion!